### PR TITLE
Add unit testing

### PR DESCRIPTION
**Pull request by alhazen11**
Screen Shot 2021-09-25 at 23 49 10

### CODE REVIEW

File: lib/services/projectService.js [hunk 1]
```diff
@@ -1,58 +1,60 @@
+require('node-absolute-path');
+
```
Comment (retzd-tech) [inline lib/services/projectService.js]: remove empty line, grouping import
```js
require('node-absolute-path');
const inquirer = require("inquirer");
```

File: test/lib/services/projectService.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+test('should do a partial mock', () => {
+    const createNewProjectResult = createNewProject(["create", "-n", "name"]);
+    expect(createNewProjectResult).toBe('mocked baz');
+    expect(createNewProject).toHaveBeenCalled();
+    const runProjectResult = runProject(["run"]);
+    expect(runProjectResult).toBe('mocked baz');
+    expect(runProject).toHaveBeenCalled();
+});
+
+let path = require('path');
```
Comment (retzd-tech) [inline test/lib/services/projectService.test.js]: Why not put this import in the top of the file ?

Comment (alhazen11) [reply test/lib/services/projectService.test.js]: sorry, it should be deleted

File: lib/services/projectService.js (hunk 1 above)
Comment (alhazen11) [reply lib/services/projectService.js]: oke

**Pull request closed without merging**